src/app-layout/index.tsx

```diff
@@ -46,7 +53,7 @@ const AppLayout = React.forwardRef(
     };
 
     // This re-builds the props including the default values
-    const props = { contentType, headerSelector, footerSelector, ...rest, ariaLabels };
+    const props = { ariaLabels, contentType, headerSelector, footerSelector, navigationWidth, toolsWidth, ...rest };
```

Comment:
Moved `ariaLabels` here to the first position to avoid accidental overrides with the `rest` object

Reply:
Found a reason why it worked this way. Undone the change